src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,80 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.TaskType;
+import duke.task.ToDo;
+import java.io.File;
```

Comment:
Just a suggestion, but perhaps you can leave a line in between the last duke import line and first java import line to make it neater?